superset-frontend/src/views/CRUD/welcome/EmptyState.tsx

```diff
@@ -57,11 +57,11 @@ export default function EmptyState({ tableName, tab }: EmptyStateProps) {
     SAVED_QUERIES: 'empty-queries.svg',
   };
   const mine = (
-    <span>{`No ${
-      tableName === 'SAVED_QUERIES'
-        ? t('saved queries')
-        : t(`${tableName.toLowerCase()}`)
-    } yet`}</span>
+    <span>
+      {tableName === 'SAVED_QUERIES'
+        ? t('No saved queries yet')
+        : t('No %(tableName)s yet', { tableName: tableName.toLowerCase() })}
```

Comment:
Wouldn't tableName need to be translated, too?

Also, L141-145 (See all) need a literal as well

Reply:
Oh right, now I understand that variable (I thought it was a regular table name in a database and didn't quite understand the point of that sentence 😆). I'll add proper typing and translations for this